packages/utils/test/getTemplate.test.ts

```diff
@@ -86,4 +86,19 @@ describe('getTemplate', () => {
       expect(getTemplate<typeof name>(name, registry, uiOptions)).toBe(CustomTemplate);
     });
   });
+  it('returns the template from registry using uiOptions key when available', () => {
```

Comment:
Consider adding an additional test that uses a name that is not one of the standard names? I.e. if a user adds their own named template, then they should be able to get it out via that name, correct?